src/enum/SubscribedMessageHttpResponse.enum.ts

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * Http Response Status to be sent back to dapr runtime as subscription response, as per 
+ * https://docs.dapr.io/reference/api/pubsub_api/#expected-http-response
+ */
+export enum SubscribedMessageHttpResponse {
```

Comment:
Hey Deepanshu, small typo - 
```suggestion
export enum SubscribeMessageHttpResponse {
```
or this one, based on the proto (preferably this, sorry for not suggesting this earlier)
```suggestion
export enum TopicEventHttpResponse {
```

Reply:
That is intentional actually. It made better understanding for me: SubscribedMessageHttpResponse instead of SubscribeMessageHttpResponse.

Reply:
It does convey - message from "subscribed" event. Whereas we send a different response when you actually subscribe to a a route/topic, this is from the callback handler. But I am okay with this too.